clone_repos.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Clones server
+[[ -d wtt_server ]] || git clone https://github.com/waterthetrees/wtt_server.git
+
+# Clones frontend
+[[ -d wtt_front ]] || git clone https://github.com/waterthetrees/wtt_front.git
+
+# Clones db
+[[ -d wtt_db ]] || git clone https://github.com/waterthetrees/wtt_db.git
```

Comment:
Great for a new pull.  How will pull before check-in be accomplished?

Reply:
Not sure if we can write a script to fully automate subsequent pulls. There can be a simple script that pulls from remote main to local main. But I'm thinking that devs will need to manually commit or stash their work on their working branch and checkout to local main anyway. So the script would only save them having to write `git pull` after all that.